request-server.go

```diff
@@ -188,30 +192,30 @@ func (rs *RequestServer) packetWorker(
 				rpkt = statusFromError(pkt, syscall.EBADF)
 			} else {
 				request = NewRequest("Stat", request.Filepath)
-				rpkt = request.call(rs.Handlers, pkt)
+				rpkt = request.call(rs.Handlers, pkt, rs.pktMgr.alloc, orderID)
```

Comment:
The caller should be able to call `pkt.orderID()` itself, right?

Reply:
pkt is an orderedRequest and so has orderID method, `request.call` requires a requestPacket that cannot call orderID itself